static/bidder-info/smartadserver.yaml

```diff
@@ -1,6 +1,6 @@
 endpoint: "https://ssb-global.smartadserver.com"
 maintainer:
-  email: "support@smartadserver.com"
+  email: "support@equativ.com"
```

Comment:
Sent email to above email address. Please reply with "received" as response.

Reply:
Based on the email thread discussion, it appears that the correct email address is `partner-integration@equativ.com`. If this is indeed the case, kindly update the contact details in the PR.

Reply:
updated